magento/pwa-studio
# Refactor User Detail Fetching to GraphQL

## Description
See PWA-138

## Related Issue





* [[PWA-138](https://jira.corp.magento.com/browse/PWA-138)] User details should be fetched from GraphQL

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Create Account
- Go through Create Account workflow
- Verify user details are displayed correctly after successful creation (at bottom on nav bar)
2. Sign-In
- Logout and sign-in with the same account as above
- Verify you're redirected to MyAccount/Sign-out Page
- Verify user details still displayed correctly
3. Sign-out
- Sign-out
- Verify user details is replaced by signin button
4. MyAccount
- While signed in, clear browser cache while retaining local storage
- Verify user details are loading correctly on initial app render

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
